Improve Oral-B documentation (#29705)
home-assistant/home-assistant.github.io: source/_integrations/oralb.markdown

- Time - total brushing time in seconds.
- Toothbrush state - whether the toothbrush is running, idle.
- Battery - toothbrush battery percentage.

<div class='note'>

> Updates of the battery sensor require an active Bluetooth connection and relatively close proximity. If you use a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All the other sensors update with active or passive connections. 

Reviewer: This link does look like it formatted correctly in the preview

Reviewer 2: Right. Thank you for the catch. Let me fix that.

> Updates of the battery sensor require an active bluetooth connection and relatively close proximity. If you are using a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All other sensors (i.e., all sensors except the battery sensor) will update with active or passive connections. 

Reviewer 2: ```suggestion
Updates of the battery sensor require an active Bluetooth connection and relatively close proximity. If you use a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All the other sensors update with active or passive connections. 
```
